README.md

```diff
@@ -1,64 +1,54 @@
-<img src="https://ucbrise.github.io/opaque/opaque.svg" width="315" alt="Opaque">
+<img src="https://mc2-project.github.io/opaque/opaque.svg" width="315" alt="Opaque">
 
 **Secure Apache Spark SQL**
 
-[![Build Status](https://travis-ci.org/ucbrise/opaque.svg?branch=master)](https://travis-ci.org/ucbrise/opaque)
+[![Build Status](https://travis-ci.org/mc2-project/opaque.svg?branch=openenclave)](https://travis-ci.org/mc2-project/opaque)
```

Comment:
Should we change the branch back to master? The travis build status should be monitoring master once this PR has been merged.